src/agent/rustjail/src/mount.rs

```diff
@@ -668,6 +668,45 @@ fn parse_mount(m: &Mount) -> (MsFlags, String) {
     (flags, data.join(","))
 }
 
+// Use secure_join to get full path of container rootfs
```

Comment:
Paths are awkward and handling can be error prone so please can you add a little more detail here? Something like:

```rust
// This function constructs a canonicalized path by combining the `rootfs` and `unsafe_path` elements.
// The resulting path is guaranteed to be ("below" / "in a directory under") the `rootfs` directory.
//
// Parameters:
//
// - `rootfs` is the absolute path to the root of the containers root filesystem directory which must exist.
// - `unsafe_path` is path inside a container. It is unsafe since it may try to "escape" from the containers
//    rootfs by using one or more "../" path elements.
```

Also, needs explanation of what happens if `unsafe_path` does not exist (what is returned?)

Reply:
Thanks for point it out in advance. I'll update it later.